pythran/pythonic/numpy/random/bytes.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+#define PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+
+#include "pythonic/include/numpy/random/bytes.hpp"
+#include "pythonic/include/numpy/random/generator.hpp"
+
+#include "pythonic/types/str.hpp"
+#include "pythonic/utils/proxy.hpp"
+
+#include <random>
+#include <string>
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+
+      types::str bytes(long length)
+      {
+        // dummy init + rewrite is faster than reserve and push_back
+        types::str result(std::string(length, 0));
```

Comment:
great to keep the comment! In my memories, resizing is slightly different from initialization with a value as you do now, but I'm ok with this anyway.